src/storage/azure.js

```diff
@@ -37,19 +37,42 @@ const downloadFileAsJson = ({ blobServiceClient }) => async (
   bucketName,
   fileName
 ) => {
-  const downloadResponse = await blobServiceClient
-    .getContainerClient(bucketName)
-    .getBlockBlobClient(fileName)
-    .download(0);
   try {
-    return await streamToString(
-      downloadResponse.readableStreamBody
-    ).then((data) => JSON.parse(data));
+    return await downloadFileAsString({blobServiceClient})(bucketName, fileName)
```

Comment:
@marcohyro a few nitpicks for you:  is this await redundant here?

consider adding the `no-return-await` eslint rule to the repo

Reply:
I am actually not exactly sure about the operator precedence because of the double call.